src/PayID/pay-id-admin-client.ts

```diff
@@ -0,0 +1,59 @@
+import PayIDError from './pay-id-error'
+
+/* eslint-disable @typescript-eslint/require-await */
+// TODO(keefertaylor): Enable lint rules when method is implemented.
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable class-methods-use-this */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable max-classes-per-file */
+
+/**
+ * An admin for Xpring's PayID service.
+ *
+ * Admin clients can update address mappings. To resolve PayIDs to addresses, please use `PayIDClient`.
+ *
+ * Inputs to this class are always in X-Addresses.
+ * @see https://xrpaddress.info/
+ *
+ * @warning This class is experimental and should not be used in production applications.
+ * TODO(keefertaylor): Export this class in index.ts when it's ready for external consumption.
+ */
+export default class PayIDAdminClient {
+  private static putEndpoint = '/put'
```

Comment:
An `add`/`POST` creates a new "payment pointer account" in the PayID service.

An `update/PUT` can CRUD addresses associated with that account _or_ the payment pointer associated with that account. So if I change my `xpringID` from `hbergren` to `hansbergren`, that looks like updating my `payment_pointer` using a `PUT` request rather than deleting my old PayID service account and creating a new one with all new mappings.

Does that sound reasonable?